docs/docs/latest/topics/using/edgectl/edge-control.md

```diff
@@ -26,327 +27,161 @@ chmod a+x edgectl
-kind: Deployment
-metadata:
-  name: telepresence-proxy
-  namespace: ambassador
-  labels:
-    app: telepresence-proxy
-spec:
-  replicas: 1
-  selector:
-    matchLabels:
-      app: telepresence-proxy
-  template:
-    metadata:
-      labels:
-        app: telepresence-proxy
-    spec:
-      containers:
-      - name: telepresence-proxy
-        image: __AES_IMAGE__   # Replace this
-        command: [ "traffic-manager" ]
-        ports:
-          - name: sshd
-            containerPort: 8022
-        env:
-          - name: AMBASSADOR_LICENSE_FILE
-            value: /.config/ambassador/license-key
-        volumeMounts:
-          - mountPath: /tmp/ambassador-pod-info
-            name: ambassador-pod-info
-          - mountPath: /.config/ambassador
-            name: ambassador-edge-stack-secrets
-            readOnly: true
-      restartPolicy: Always
-      terminationGracePeriodSeconds: 0
-      volumes:
-      - downwardAPI:
-          items:
-          - fieldRef:
-              fieldPath: metadata.labels
-            path: labels
-        name: ambassador-pod-info
-      - name: ambassador-edge-stack-secrets
-        secret:
-          secretName: ambassador-edge-stack
-``` 
-
-6. Apply them:
+If `/etc/resolv.conf` is correct, but you have a local DNS server available on 10.0.0.1 that should be used for non-cluster queries, you could run Configure fallback server:
 
 ```bash
-$ kubectl apply -f traffic-manager.yaml
-service/telepresence-proxy created
-deployment.apps/telepresence-proxy created
+$ sudo edgectl daemon --fallback 10.0.0.1
+Launching Edge Control Daemon v1.0.0-ea5 (api v1)
 ```
 
-### Traffic Agent
-
-Any microservice running in a cluster with a traffic manager can opt in to intercept functionality by including the Traffic Agent in its pods. 
-
-1. Since the Traffic Agent is built on Ambassador Edge Stack, it needs the same RBAC permissions that Ambassador does. The easiest way to provide this is to create a `ServiceAccount` in your service's namespace, bound to the `ambassador` `ClusterRole`:
-
-```yaml
----
-apiVersion: v1
-kind: ServiceAccount
-metadata:
-  name: traffic-agent
-  namespace: default
-  labels:
-    product: aes
----
-apiVersion: rbac.authorization.k8s.io/v1beta1
-kind: ClusterRoleBinding
-metadata:
-  name: traffic-agent
-  labels:
-    product: aes
-roleRef:
-  apiGroup: rbac.authorization.k8s.io
-  kind: ClusterRole
-  name: ambassador
-subjects:
-- kind: ServiceAccount
-  name: traffic-agent
-  namespace: default
+### `edgectl disconnect`
+
+Disconnect from the cluster.
+
+### `edgectl intercept`
+
+Intercept enables the cluster to initiate traffic to the local environment. To prevent unwanted traffic from being to to the cluster, `intercept` creates routing rules that specify which traffic to send to the local environment. An `intercept` is created on a per (Kubernetes) deployment basis. Each deployment must have a traffic agent installed in order for `intercept` to function.
```

Comment:
"To prevent unwanted traffic from being *(missing stuff)* to ~to~ the cluster,"

Missing and doubled stuff.